lib/cloudant.js

```diff
@@ -712,7 +706,10 @@ Cloudant.prototype.updateAttributes = function(model, id, data, options, cb) {
     _.mergeWith(doc, data, function(dest, src) { return src; });
     self.create(model, doc, options, function(err) {
       if (err) return cb(err);
-      return cb(err, data);
+      mo.db.get(id, function(err, doc) {
```

Comment:
any reason why function `_insert` only returns `id` but not the full `result`? I think the `result` returned from database is the best data to show the insert result, `mo.db.get` would have delay, which lead to 409 conflict if that doc get updated by other one.